backup: TestRestoreRetryFastFails failed

backup.TestRestoreRetryFastFails [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20172662?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20172662?buildTab=artifacts#/) on master @ [a5a996e24c79895ff4f9fbfc9a1e69c21a91aeb8](https://github.com/cockroachdb/cockroach/commits/a5a996e24c79895ff4f9fbfc9a1e69c21a91aeb8):


```
=== RUN   TestRestoreRetryFastFails
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/dd23324be9c345c20b5cc646a9744f1e/logTestRestoreRetryFastFails373362087
    test_log_scope.go:82: use -show-logs to present logs inline
    restore_test.go:208: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/dd23324be9c345c20b5cc646a9744f1e/logTestRestoreRetryFastFails373362087
--- FAIL: TestRestoreRetryFastFails (129.79s)
=== RUN   TestRestoreRetryFastFails/retry_policy_fast_fails_if_insufficient_progress_is_made
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    restore_test.go:182: condition failed to evaluate within 2m0s: expected job status failed, but got paused
    testutils.go:307: no Invalid Descriptors
--- FAIL: TestRestoreRetryFastFails/retry_policy_fast_fails_if_insufficient_progress_is_made (124.79s)
=== RUN   TestRestoreRetryFastFails/retry_policy_times_out_when_enough_progress_is_made
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    testutils.go:307: no Invalid Descriptors
    restore_test.go:197: 
        	Error Trace:	pkg/backup/restore_test.go:197
        	Error:      	"9" is not greater than "12"
        	Test:       	TestRestoreRetryFastFails/retry_policy_times_out_when_enough_progress_is_made
--- FAIL: TestRestoreRetryFastFails/retry_policy_times_out_when_enough_progress_is_made (4.97s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRestoreRetryFastFails.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

backup.TestRestoreRetryFastFails [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20182710?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20182710?buildTab=artifacts#/) on master @ [85cfb7b260090484c7e040c171bc7b5ac5c2cd3d](https://github.com/cockroachdb/cockroach/commits/85cfb7b260090484c7e040c171bc7b5ac5c2cd3d):


```
=== RUN   TestRestoreRetryFastFails
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/dd23324be9c345c20b5cc646a9744f1e/logTestRestoreRetryFastFails435186078
    test_log_scope.go:82: use -show-logs to present logs inline
    restore_test.go:208: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/dd23324be9c345c20b5cc646a9744f1e/logTestRestoreRetryFastFails435186078
--- FAIL: TestRestoreRetryFastFails (124.47s)
=== RUN   TestRestoreRetryFastFails/retry_policy_fast_fails_if_insufficient_progress_is_made
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    restore_test.go:182: condition failed to evaluate within 2m0s: expected job status failed, but got paused
    testutils.go:307: no Invalid Descriptors
--- FAIL: TestRestoreRetryFastFails/retry_policy_fast_fails_if_insufficient_progress_is_made (122.82s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRestoreRetryFastFails.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/backup/restore_test.go
+++ b/pkg/backup/restore_test.go
@@ -105,11 +105,11 @@ func TestRestoreRetryFastFails(t *testing.T) {
 	// Max duration needs to be long enough such that the restore job
 	// runtime does not exceed the max duration of the retry policy, or
 	// else very few attempts will be made.
-	maxDuration := 100 * time.Millisecond
+	maxDuration := 500 * time.Millisecond
 	if skip.Stress() {
 		// Under stress, the restore will take longer to complete, so we need to
 		// increase max duration accordingly.
-		maxDuration = time.Second
+		maxDuration = 1500 * time.Millisecond
 	}
 	const numAccounts = 10
 

